### PR TITLE
Added new file locations to check

### PR DESCRIPTION
**Pull request by Zach-Attach**
Removed redundant file checks
Added new filechecks to reflect checks in https://github.com/GantMan/jail-monkey

### CODE REVIEW

File: src/ios/IRoot.m [hunk 1]
```diff
@@ -136,28 +136,29 @@ - (bool) jailbroken {
 
         (f = fopen("/bin/bash", "r")) ||
         (f = fopen("/bin/sh", "r")) ||
-        (f = fopen("/usr/sbin/sshd", "r")) ||
```
Comment (WuglyakBolgoink) [inline src/ios/IRoot.m]: @Zach-Attach Why we should remove this check?

Comment (Zach-Attach) [reply src/ios/IRoot.m]: @WuglyakBolgoink Sorry, that wasn't intentional. I saw "/usr/bin/sshd" and thought it was a duplicate.

Comment (Zach-Attach) [reply src/ios/IRoot.m]: @WuglyakBolgoink It is a duplicate. See line 116

Comment (WuglyakBolgoink) [reply src/ios/IRoot.m]: ah. ok.

File: src/ios/IRoot.m [hunk 2]
```diff
@@ -136,28 +136,29 @@ - (bool) jailbroken {
...
         (f = fopen("/Applications/MxTube.app", "r")) ||
-
         (f = fopen("/Applications/IntelliScreen.app", "r")) ||
         (f = fopen("/Applications/FakeCarrier.app", "r")) ||
+
         (f = fopen("/Applications/blackra1n.app", "r")) ||
         (f = fopen("/Applications/IntelliScreen.app", "r")) ||
-        (f = fopen("/Applications/FakeCarrier.app", "r"))
+        (f = fopen("/Applications/FakeCarrier.app", "r")) ||
+        (f = fopen("/usr/bin/frida-server", "r")) ||
```
Comment (WuglyakBolgoink) [inline src/ios/IRoot.m]: @Zach-Attach good check, to avoid runtime code-injection

**Comment by WuglyakBolgoink** [comment]
thank you @Zach-Attach

**Pull request merged**